packages/grafana-data/src/types/action.ts

```diff
@@ -10,6 +10,7 @@ export enum ActionType {
 export interface Action {
   type: ActionType;
   title: string;
+  confirmation?: string;
```

Comment:
naming? also probably will support markdown so users can highlight the interpolated parts of the message?